File: VB/NET40/WhatIsMyBrowser.CommonTypesVB/UserAgentDatabaseDump.vb
Imports Newtonsoft.Json

Public Class UserAgentDatabaseDump
    <JsonProperty("url")>
    Public Property Url As Uri

    <JsonProperty("num_of_useragents")>
    Public Property NumberOfUserAgents As Long

    <JsonProperty("file_format")>
    Public Property FileFormat As String

    <JsonProperty("sha_sum_256")>
    Public Property ShaSum256 As String

    <JsonProperty("created_at")>
    Public Property CreatedAt As DateTimeOffset
End Class

File: CS/NET40/WhatIsMyBrowser.CommonTypes/UserAgentDatabaseDump.cs
using System;
using Newtonsoft.Json;

namespace WhatIsMyBrowser.CommonTypes
{
    public class UserAgentDatabaseDump
    {
        [JsonProperty("url")]
        public Uri Url { get; set; }

        [JsonProperty("num_of_useragents")]
        public long NumberOfUserAgents { get; set; }

        [JsonProperty("file_format")]
        public string FileFormat { get; set; }

        [JsonProperty("sha_sum_256")]
        public string ShaSum256 { get; set; }

        [JsonProperty("created_at")]
        public DateTimeOffset CreatedAt { get; set; }
    }
}